instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -272,11 +272,13 @@ def _instrumented_lambda_handler_call(
             lambda_event, event_context_extractor
         )
 
-        span_kind = None
```

Comment:
I believe this was added here for a reason, but I don't really recall why.

Reply:
I deleted it because the `span_kind` variable from that line because it is being initialised anyway in the try/except statement below.

Reply:
I see that, but what I was trying to say is this was added for a reason, IIRC if there is an exception that is not one from the list of exceptions in the catch block, it raises `NameError`.

Reply:
I put it back.